trussworks/react-file-viewer
# [MB-11016] Update three dependency to resolve security vulnerability

The three library has a security vulnerability that is only fixed in version 0.137, so this upgrades us to that version.  The corresponding MyMove PR can be found here https://github.com/transcom/mymove/pull/8095.

To test this you can run:
`yarn install`
`yarn start`

Visit localhost:8081 in your browser and see that the document viewer renders the pdf.

I tested other image types by manually editing the app.js sample to use jpg, gif, and png.

https://github.com/transcom/mymove/security/dependabot/yarn.lock/three/open
